spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -207,21 +210,20 @@ def get_dtcm_usage_in_bytes(self):
 
     def _get_synapse_params_size(self):
         return (_SYNAPSES_BASE_SDRAM_USAGE_IN_BYTES +
-                (4 * self.__n_synapse_types))
+                (_WORD_SIZE * self.__n_synapse_types))
 
     def _get_static_synaptic_matrix_sdram_requirements(self):
 
         # 4 for address of direct addresses, and
         # 4 for the size of the direct addresses matrix in bytes
-        return 8
+        return 1 * _WORD_SIZE
```

Comment:
This was returning 8, and is now returning 4... ?  (The comment above it needs to be updated, at least...)

Reply:
I wonder why I did that...!

Reply:
Looks like it was just a lapse of concentration and a slip of the fingers.